MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include LICENSE
-include *.md
\ No newline at end of file
+include *.md
+include *.toml
```

Comment:
I'm not sure if we need to include all `.toml` files, this will include the `pyproject.toml` which isn't needed for the package. If deepsource needs the `.deepsource.toml` as part of the generated package I think it would be better to explicitly include it.

Reply:
Thank you for your reply. Should I remove the changes from `MANIFEST.in` or write `include .deepsource.toml` ?

Reply:
Looking at https://deepsource.io/docs/config/deepsource-toml.html#version I believe the file just has to live in the repository root, not in the generated package. If you could test removing the `include *.toml` line entirely in pnijhara/hug and verify that deepsource still runs correctly I'd rather leave the `MANIFEST.in` file alone in this repo

Reply:
DeepSource does not need to be included in `MANIFEST.in`. I think it is better to remove it.